lib/improver/tests/ensemble_copula_coupling/ensemble_copula_coupling/test_GeneratePercentilesFromProbabilities.py

```diff
@@ -36,7 +36,9 @@
 import numpy as np
 import unittest
 
+import cf_units as unit
 from iris.cube import Cube
+from iris.coords import DimCoord
 from iris.tests import IrisTest
 
```

Comment:
Under class Test_process, do you need a unit test to check that it raises an error if both percentiles and no_of_percentiles are set?

Reply:
I think this is covered in  test_check_data_over_specifying_percentiles()
Perhaps has been added since.

Reply:
:white_check_mark: